eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/http/HttpAuthenticationComposite.java

```diff
@@ -266,70 +267,70 @@ private void updateControls(boolean isBasic, boolean isNtlm) {
     
     private void bindControls(final DataBindingContext context) {
         final EditingDomain domain = AdapterFactoryEditingDomain.getEditingDomainFor(getTargetObject());
-        final Realm realm = SWTObservables.getRealm(_authTypeCombo.getCombo().getDisplay());
+        final Realm realm = DisplayRealm.getRealm(_authTypeCombo.getCombo().getDisplay());
 
-        _bindingValue = new WritableValue(realm, null, HTTPBindingType.class);
-        final IObservableValue authType = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthUser = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthPwd = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthRealm = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthHost = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthPort = new WritableValue(realm, null, String.class);
-        final IObservableValue ntlmAuthDomain = new WritableValue(realm, null, String.class);
+        _bindingValue = new WritableValue<>(realm, null, HTTPBindingType.class);
+        final IObservableValue<String> authType = new WritableValue<String>(realm, null, String.class);
```

Comment:
I mentioned it only on previous line before but you can write new WritableValue<String> also for all these lines.

When using SonarLint this kind of improvement are reported.